backend/services/healthCheck.ts

```diff
@@ -13,9 +15,9 @@ const gatherCheckInfo = (total, currentValue) => ({ ...total, [currentValue.name
 
 const getBuild = () => {
   try {
-    // eslint-disable-next-line import/no-unresolved,global-require
-    return require('../../build-info.json')
+    return JSON.parse(fs.readFileSync(path.join(__dirname, '../../../build-info.json'), 'utf-8'))
```

Comment:
*non blocking*
Perhaps using `process.cwd()` might be better over `__dirname`. It would make sure that the file is always located from wherever the `npm run start` command was run on the path.

Reply:
Good shout!